Revert reverting Electron 19 upgrade

sergiou87 opened the pull request:
## Description

With #14861 I forgot to revert the change in Electron before merging the branch 🤦 Thanks @tidy-dev for letting me know 🙇‍♂️ 

Should we add the workaround @niik came up with for the Dev Tools crash? Only for dev builds of course.

## Release notes

Notes: no-notes

sergiou87 committed:
Revert "Revert "Merge pull request #14813 from desktop/upgrade-to-electron-v19""

This reverts commit 1bfcd1986df982bae94349c064833e33f8730254.

tidy-dev commented:
> workaround niik came up with for the Dev Tools crash? Only for dev builds of course.

Seeing as it is only for dev builds, I am not opposed to that.

sergiou87 committed:
Add workaround for an Electron 19 crash

This crash happens when attempting to write anything in the Dev Tools console.
See https://github.com/electron/electron/issues/34350

sergiou87 commented:
Cool! Added the change [0c25285](https://github.com/desktop/desktop/pull/14885/commits/0c25285238e894762fcd0747e2d469b9a5309add)

EDIT: Cmd + Enter closed the PR for some reason O_o

tidy-dev reviewed (approved):
✨
